Source/Scene/Scene.js

```diff
@@ -2696,6 +2706,68 @@ define([
         return object;
     };
 
+    var scratchPickDepthPassState;
+
+    function renderForPickDepth(scene, drawingBufferPosition) {
+        var context = scene._context;
+        var us = context.uniformState;
+        var frameState = scene._frameState;
+
+        // Update with previous frame's number and time, assuming that render is called before picking.
+        updateFrameState(scene, frameState.frameNumber, frameState.time);
+        frameState.cullingVolume = getPickCullingVolume(scene, drawingBufferPosition, 1, 1);
+        frameState.passes.render = true;
```

Comment:
Is it better for this to be
```javascript
frameState.passes.pick = true;
```
Or to do whatever the shadow map pass does since they only writes depth.

If doc, please add a comment.